app/modules/intelligence/agents_copy/chat_agents/langchain_agent.py

```
@@ -0,0 +1,182 @@
+
+import agentops
+from app.modules.intelligence.provider.provider_service import (
+    ProviderService,
+    AgentType,
+)
+from .crewai_agent import AgentConfig, TaskConfig
+from app.modules.utils.logger import setup_logger
+from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
+from ..chat_agent import ChatAgent, ChatAgentResponse, ChatContext
+from pydantic import BaseModel
+import json
```

Ruff (0.8.2)

Redefinition of unused `json` from line 1

Remove definition: `json`

(F811)